components-styled/StyledIconButtonwrapper.jsx

```diff
@@ -3,15 +3,10 @@ import styled from 'styled-components';
 const StyledIconButtonWrapper = styled.div`
   display: flex;
   justify-content: space-evenly;
+  align-self: end;
   gap: 1rem;
   height: 40px;
```

Comment:
Morten would say stick to either rem or pixel consequently :)
Felix would say stick to pixel only. Because if user changes rem for accessibility reasons it will ruin your layout :)
I think it's fine :)

Reply:
For example, I prefer pixels in gaps, because they are independent of whether the view changes or not. :)